docs/components/dartpad/menus/exposed_dropdown/main.dart

```diff
@@ -0,0 +1,69 @@
+import 'package:flutter/material.dart';
+
+void main() => runApp(MyApp());
+
+class MyApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'Exposed Menu Demo',
+      debugShowCheckedModeBanner: false,
+      home: ExposedDropdownMenuDemo(),
```

Comment:
I think we should set the `ThemeData.primarColor` here to be purple.

Reply:
I didn't do it for this example but I did it for the dropdown example. That way it matches the demo.